src/libs/ajax/metrics/useMetrics.ts

```diff
@@ -0,0 +1,13 @@
+import { useMemo } from 'react'
+import { Ajax } from 'src/libs/ajax'
+
+
+export type CaptureEventFn = (event: string, details?: {}) => void
+
+export const useMetricsEvent = (): CaptureEventFn => {
+  const sendEvent = useMemo(() => Ajax().Metrics.captureEvent, [])
+  const captureEvent: CaptureEventFn = (event: string, details?: {}): void => {
+    void sendEvent(event, details)
+  }
+  return captureEvent
```

Comment:
In the spirit of "delaying decisions", would it make sense to return an object `{ captureEvent }` instead of just `captureEvent` to allow adding additional methods/properties later?

Reply:
i like it... since more of the metrics contract could come over into the hook at some point.